src/components/apps/Calendar/Views/MonthView.module.scss

```diff
@@ -0,0 +1,64 @@
+.monthView {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: 30px 1fr 1fr 1fr 1fr 1fr 1fr;
+  flex-grow: 1;
+  font-size: 0.8rem;
+
+  > div {
+    padding: 0.5rem;
+    text-align: right;
+  }
+
+  .weekday,
+  .weekend {
+    border-bottom: 1.5px solid #c7c7cc;
+  }
+
+  .weekend {
+    color: #808080;
+  }
+
+  .day {
+    border: 1px solid #e6e5e6;
+    border-top: none;
+
+    &:nth-child(7n) {
+      border-right: none;
+    }
+
+    &:nth-child(1n) {
+      border-left: none;
+    }
+
+    &:nth-child(n + 43) {
+      border-bottom: none;
+    }
+
+    &:nth-child(7n),
+    &:nth-child(7n-1) {
+      background-color: #f5f5f5;
+    }
+
+    .dateNumber {
+      height: 1.5em;
+      width: 1.5em;
+      line-height: 1.5em;
+      border-radius: 50%;
+      display: inline-block;
+      text-align: center;
+      color: #c0c0c0;
```

Comment:
Better that this line be `var(--app-color-dark)`
![image](https://user-images.githubusercontent.com/47742487/115970114-13a4ac00-a55e-11eb-9ded-2b2fbcf71d87.png)

Reply:
Done